src/Components/Patient/SampleTestCard.tsx

```diff
@@ -126,7 +126,7 @@ export const SampleTestCard = (props: SampleDetailsProps) => {
               Result{" "}
             </div>
             <div className="mt-1 overflow-x-scroll whitespace-normal break-words text-sm font-medium leading-5">
-              {_.startCase(_.camelCase(itemData.result))}
+              {startCase(camelCase(itemData.result))}
```

Comment:
Why not use css to achieve the same result; 
https://tailwindcss.com/docs/text-transform#transforming-text